src/FileIO/SidreFileIO.cc

```diff
@@ -136,6 +136,30 @@ SidreFileIO::SidreFileIO(const std::string fileName, AccessType access):
   ENSURE(mFileOpen && mDataStorePtr != 0);
 }
 
+//------------------------------------------------------------------------------
+// Construct with a set number of restart files.
+//------------------------------------------------------------------------------
+SidreFileIO::SidreFileIO(int numFiles):
+  FileIO(),
+  mDataStorePtr(0)
+{
+  if (numFiles > 0 && numFiles <= Process::getTotalNumberOfProcesses())
+    numRestartFiles = numFiles;
+}
```

Comment:
I might be missing it, is there anywhere we are using this constructor, or a instance we might use it? Or is this an overload of a base FileIO constructor?

Reply:
It's not being used anywhere, but I added both constructors if needed for future use somewhere. I can remove and it can be added later if needed.

Reply:
Okay lets go ahead and remove it provided it doesn't break anything to do so.